components/EmojiPicker/EmojiRowList.tsx

```diff
@@ -27,6 +27,9 @@ interface EmojiRowListProps {
 
 const keyExtractor = (_: unknown, index: number) => String(index);
 
+// Works around issue with Android not picking up scrolls
+const ListRenderer = Platform.OS === "ios" ? ReanimatedFlashList : FlatList;
```

Comment:
Yeah I also had many issues with FlashList on Android

Reply:
There's definitely a trade off with it unfortunately, we might want to keep a tracking list to make better decisions about if we feel it's worthwhile keeping it